complete/src/main/java/com/example/restservice/dao/Queue.java

```diff
@@ -30,13 +28,13 @@
   @GenericGenerator(name = "uuid2", strategy = "uuid2")
   private String queueId;
 
-  private String userId;
+  private String ownerId;
 
   @OneToMany(cascade = CascadeType.ALL, mappedBy = "queue")
   private List<User> users;
 
-  public Queue(String queueName, String userId) {
+  public Queue(String queueName, String ownerId) {
     this.queueName = queueName;
-    this.userId = userId;
+    this.ownerId = ownerId;
```

Comment:
I'll follow up with a "Token" Endpoints PR, and the plan is to have a ownerId for both token and queue objects.